modules/cluster/addons.tf

```diff
@@ -25,28 +25,26 @@ module "critical_addons_node_group" {
 
 data "aws_region" "current" {}
 
-# The kube-proxy EKS addon introduced regressions of #124 and #209. We will move to the EKS addon when these are fixed.
-module "kube_proxy" {
-  source = "./kubectl"
-  config = local.config
-  manifest = templatefile(
-    "${path.module}/addons/kube-proxy.yaml",
-    { aws_region = data.aws_region.current.name },
-  )
-}
 // When upgrading k8s version run `aws eks describe-addon-versions --kubernetes-version <version>` to get addon_version numbers
 
 resource "aws_eks_addon" "vpc-cni" {
   cluster_name      = local.config.name
   addon_name        = "vpc-cni"
-  addon_version     = "v1.10.2-eksbuild.1"
+  addon_version     = "v1.11.0-eksbuild.1"
+  resolve_conflicts = "OVERWRITE"
+}
+
+resource "aws_eks_addon" "kube-proxy" {
```

Comment:
Have you verified that [this](https://github.com/cookpad/terraform-aws-eks/commit/dc5394ac5f253eb3e528cb6ad8bd6471bd075c1c) has been resolved?

Reply:
In theory has been fixed in 1.22 https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html but I'll check if the configmap of kube-proxy binding metricsBindAddress is now to 0.0.0.0 instead of 127.0.0.1

Reply:
I have created with terraform apply the example cluster that is in the /example folder of this repo and the bindaddress of the configmap "kube-proxy-config" is now 0.0.0.0, so the kube-proxy addon issue was fixed.